ccdproc/combiner.py

```diff
@@ -72,6 +109,12 @@ class Combiner:
         description. If ``None`` it uses ``np.float64``.
         Default is ``None``.
 
+    xp : array namespace, optional
+        The array namespace to use for the data. If not provided, it will
+        be inferred from the first `~astropy.nddata.CCDData` object in
+        ``ccd_iter``.
+        Default is ``None``.
```

Comment:
```suggestion
    xp : array namespace, optional
        The array namespace to use for the data. If `None` or not provided, it will
        be inferred from the first `~astropy.nddata.CCDData` object in
        ``ccd_iter``.
        Default is `None`.
```